src/libraries/Common/src/Interop/Windows/Kernel32/Interop.LocalAlloc.cs

```diff
@@ -8,17 +8,17 @@ internal static partial class Interop
 {
     internal static partial class Kernel32
     {
-        internal const uint LMEM_FIXED = 0x0000;
-        internal const uint LMEM_MOVEABLE = 0x0002;
-        internal const uint LMEM_ZEROINIT = 0x0040;
+        private const uint LMEM_FIXED = 0x000;
+        private const uint LMEM_ZEROINIT = 0x0040;
+        private const uint LPTR = LMEM_FIXED | LMEM_ZEROINIT;
 
         [LibraryImport(Libraries.Kernel32)]
-        internal static partial IntPtr LocalAlloc(uint uFlags, nuint uBytes);
+        private static partial nint LocalAlloc(uint uFlags, nuint uBytes);
```

Comment:
```suggestion
        private static partial void* LocalAlloc(uint uFlags, nuint uBytes);
```

There is no reason this can't use pointers directly.

Reply:
I was thinking the P/Invoke should correspond with the actual API, which is defined:

```cpp
DECLSPEC_ALLOCATOR HLOCAL LocalAlloc(
  [in] UINT   uFlags,
  [in] SIZE_T uBytes
);
```

Because a `HLOCAL` is a handle we cannot obtain a pointer by casting - unless the  `LMEM_FIXED` flag is specified. Otherwise, documentation states a pointer must be obtained with the `LocalLock` function.pointer.

Reply:
An `HLOCAL` and underlying `HANDLE` type are all `void*` types. Is the intent here simply to avoid direct pointer declaration? I'm dubious this has a lot of value, but if you feel strongly about it feel free to obscure the type and add a comment as to why.